test/Microsoft.AspNetCore.Mvc.Core.Test/FileStreamResultTest.cs

```diff
@@ -45,11 +48,127 @@ public void Constructor_SetsContentTypeAndParameters()
             var result = new FileStreamResult(stream, contentType);
 
             // Assert
+            Assert.False(result.EnableRangeProcessing);
             Assert.Equal(stream, result.FileStream);
             MediaTypeAssert.Equal(expectedMediaType, result.ContentType);
         }
 
         [Fact]
+        public void Constructor_SetsRangeProcessingParameters()
+        {
+            // Arrange
+            var stream = Stream.Null;
+            var contentType = "text/plain";
+            var expectedMediaType = contentType;
+            var lastModified = new DateTimeOffset();
+            var entityTag = new EntityTagHeaderValue("\"Etag\"");
+
+            // Act
+            var result = new FileStreamResult(
+                fileStream: stream,
+                contentType: new MediaTypeHeaderValue(contentType),
+                enableRangeProcessing: true,
+                lastModified: lastModified,
+                entityTag: entityTag);
+
+            // Assert
+            Assert.True(result.EnableRangeProcessing);
+            Assert.Equal(lastModified, result.LastModified);
+            Assert.Equal(entityTag, result.EntityTag);
+            MediaTypeAssert.Equal(expectedMediaType, result.ContentType);
+        }
+
+        [Theory]
+        [InlineData(0, 4, "Hello", 5)]
+        [InlineData(6, 10, "World", 5)]
```

Comment:
https://tools.ietf.org/html/rfc7233#section-2.1 ..Look for the examples in these

```
Additional examples, assuming a representation of length 10000:

   o  The final 500 bytes (byte offsets 9500-9999, inclusive):
        bytes=-500
   Or:
        bytes=9500-
```